src/cmd/pkg/pagination/list_test.go

```diff
@@ -0,0 +1,89 @@
+package pagination
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func mockListFn(p Page) ([]string, string, error) {
```

Comment:
can you add a test that doesn't set `Page.Size`? let it default to zero.

Reply:
Done